C#Backend/InpatientTherapySchedulingProgram/Controllers/UserController.cs

```diff
@@ -33,6 +33,21 @@ public async Task<ActionResult<IEnumerable<User>>> GetUser()
             return Ok(allUsers);
         }
 
+<<<<<<< HEAD
```

Comment:
Looks like there are some merge conflicst

Reply:
Ah, my bad.. that's the backend. I only check the frontend. I'll keep the current changes that are on master.